IFC4x3/Properties/f/FaultActivity_2JUec9o7r0AvVBR7Pu457g/Documentation.md

```diff
@@ -0,0 +1 @@
+A fault that has slipped in historic time and which is likely to slip again in the future.
```

Comment:
This seems to be a description of a fault and not the property. What is supposed to be written into this property? The datatype is `IfcText` so I would expect a much tighter description.

Reply:
Need input from @j-weil and the domain team on this!

Reply:
Ping @j-weil

Reply:
We recommend to extend the description as follows: 
"Qualitative rating of the fault activitity (active/inactive/...)  A fault that has slipped in historic time and which is likely to slip again in the future." IfcText or Ifc label would be OK, no predfined enumerations as many ratings could be used.